src/AbstractExtendedPdo.php

```diff
@@ -912,6 +912,10 @@ protected function bindValue(PDOStatement $sth, $key, $val)
         }
 
         if (is_bool($val)) {
+            if($this->parser instanceof Parser\MysqlParser) {
```

Comment:
Add whitespace `if ($this->parser instanceof ...` on it. For PSR's sake.

Reply:
Whoops! Thanks for the revision.